docs/README.md

```diff
@@ -149,6 +167,8 @@ forc index check
 +--------+------------------------+------------------------------------------------------------------+
 ```
 
+If there are any missing components please install them in the [Dependencies](#dependencies) section. 
```

Comment:
So some of these components being missing is OK for the Quickstart (e.g., `psql`, `fuel-core`, etc), but some are required (e.g., `docker`, `fuelup`) - so we (indexer) should make this clearer in a follow up PR.

Reply:
I will go ahead and remove this for now then but I think it would be valuable to show the output in the `2.1 Check for components` section only showing what is required rather than all components installed in that follow up PR👍